src/test/java/net/minecraftforge/test/CriticalHitEventTest.java

```diff
@@ -0,0 +1,34 @@
+package net.minecraftforge.test;
+
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.entity.player.CriticalHitEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.Mod.EventHandler;
+import net.minecraftforge.fml.common.event.FMLInitializationEvent;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+import net.minecraftforge.fml.common.eventhandler.Event.Result;
+
+@Mod(modid="criticalhiteventtest", name="CriticalHitEventTest", version="0.0.0")
+public class CriticalHitEventTest
+{
+    public static final boolean ENABLE = false;
+
+    @EventHandler
+    public void init(FMLInitializationEvent event)
+    {
+        MinecraftForge.EVENT_BUS.register(this);
+    }
+
+    @SubscribeEvent
+    public void onCriticalHit(CriticalHitEvent  event)
+    {
+        if(ENABLE)
+        {
+            event.setResult(Result.ALLOW); //Every hit is Critical
+            //event.setResult(Result.Deny);//No hit will be Critical
+            //event.setResult(Result.Default); //Vanilla Hits
+            System.out.println(event.getTarget() + " got hit by " + event.getEntityPlayer());
```

Comment:
In that case, I would specify that in the documentation for the event (honestly the documentation right now is fairly useless and should be improved as well: "getDamageModifier returns the damage modifier" is not useful information) and set the priority for the test event to lowest.